### PR TITLE
Feat/translation-bubble

### PR DESCRIPTION
**Pull request by mrifkiw**

### CODE REVIEW

**Review by yrojx — CHANGES_REQUESTED** [review]
3 files of Bubble Chat can be into 1 file and keep the code less. The keyword is "conditional modifiers"

**Comment by mrifkiw** [comment]
Bubble chat file have been refactor into one file, please check my last push.

File: Satuju/Module/TranslationBubble/View/TranslationBubble.swift [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            .frame(maxWidth: 342, minHeight: 156)
+            .background(Color.white)
+            .clipShape(
+                RoundedRectangle(
+                    cornerRadius: 12
+                )
+            )
+        .shadow(color: .black.opacity(0.25), radius: 10, x: 0, y: 6)
+        }
+        .flipsForRightToLeftLayoutDirection(false)
```
Comment (yrojx) [inline Satuju/Module/TranslationBubble/View/TranslationBubble.swift:54]: unused function

File: Satuju/Module/TranslationBubble/View/TranslationBubble.swift [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                        )
+                    Divider()
+                        .background(Color("Off-Color"))
+                    Text(textTranslationResult)
+                        .font(
+                            .custom("NotoSans-Bold", size: 15)
+                        )
+                        .foregroundColor(Color("PurplePrimary"))
+                }
+                .environment(\.layoutDirection, .leftToRight)
```
Comment (yrojx) [inline Satuju/Module/TranslationBubble/View/TranslationBubble.swift:33]: can be just once declaration

**Review by yrojx — CHANGES_REQUESTED** [review]

**Review by yrojx — APPROVED** [review]

**Pull request merged**